### DISCUSSION TITLE
Add dot before fp16

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#101] [Ideas]
Please, change the prefix "fp16" to ".fp16" like there:
https://github.com/hako-mikan/sd-webui-supermerger/commit/32e9ec86850c58cba66e7aa78173718b330fc72f#diff-8094f5480d725865d6875560e308eec22f4b79c2d68ac50374cf61f7d5a119cb

**Comment by miasik** [comment]
#106

**Comment by hako-mikan** [comment]
Thanks for your cooperation!
